src/crawler.cpp

```diff
@@ -225,10 +226,9 @@ void crawler::run()
       }
     }
     QMap<QString, int>::const_iterator x;
-    for (x = parser.getKeywords().begin();
-      x != parser.getKeywords().end(); ++x)
-    {
-      add_keyword_to_db(url, x.key(), x.value());
+    auto keywords{parser.getKeywords()};
+    for (auto[key, val] : *keywords) {
+      add_keyword_to_db(url, key, val);
```

Comment:
@shaneallcroft not sure if you caught this change, but this syntax is super dope. New C++17-enabled structural binding.

And I missed the `const_iterator` above... Whole point here was to remove it, haha.